scripts/us_fda/drugs/README.md

```diff
@@ -0,0 +1,108 @@
+# US FDA Drugs Import
+## About the Dataset
+### URL
+CSV is available for download from https://www.fda.gov/media/89850/download
+### Overview
+“Drugs@FDA includes information about drugs, including biological products, approved for human use in the United States (see FAQ), but does not include information about FDA-approved products regulated by the Center for Biologics Evaluation and Research (for example, vaccines, allergenic products, blood and blood products, plasma derivatives, cellular and gene therapy products). For prescription brand-name drugs, Drugs@FDA typically includes the most recent labeling approved by the FDA (for example, Prescribing Information and FDA-approved patient labeling when available), regulatory information, and FDA staff reviews that evaluate the safety and effectiveness of the drug.”
+### License
+Drugs@FDA is public domain and made available with a Creative Commons CC0 1.0 Universal dedication.
+
+## Scripts
+### create_enums.py
+
+Functions involving creating the mcf for related enums and loading the necessary enum dictionaries can be found in `create_enums.py`.
+
+Running `create_enums.py` from command line will write the file FDADrugsEnumSchema.mcf to the current directory.
```

Comment:
Can we not check-in the generated MCFs in github? It adds to the bloat and the entire repo is being downloaded (for import automation) which is wasteful bytes on the wire.

If its not too big, we can keep the downloaded CSV. In theory, with that and the code here, we can get the MCFs and cleaned CSVs.